File: benchmark/results/java-invocation-inline/run-002/b-baseline/output/com/metaengine/demo/ordering/services/OrderValidator.java
package com.metaengine.demo.ordering.services;

import java.util.List;

import com.metaengine.demo.ordering.aggregates.Order;

/** OrderValidator for the ordering domain. */
public class OrderValidator {

    /** Validate and create a new Order from partial input. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Order by id. Returns null if not found. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Orders up to the given limit. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete the Order with the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-script-sonnet/run-002/b-baseline/output/src/domain/ordering/services/OrderValidator.ts
import { Order } from '../aggregates/Order';

/** Validator for enforcing ordering domain rules on orders. */
export class OrderValidator {
  create(input: Partial<Order>): Order {
    void input;
    throw new Error('not implemented');
  }

  findById(id: string): Order | null {
    void id;
    throw new Error('not implemented');
  }

  list(limit: number): Order[] {
    void limit;
    throw new Error('not implemented');
  }

  delete(id: string): void {
    void id;
    throw new Error('not implemented');
  }
}
